explorer/src/explorer/ExplorerOptions.hs

```diff
@@ -16,51 +16,32 @@ import           Options.Applicative.Simple (Parser, auto, help, long, metavar,
                                              showDefault, simpleOptions, strOption,
                                              switch, value)
 import           Prelude                    (show)
-import           Serokell.Util.OptParse     (fromParsec)
 
 import           Paths_cardano_sl_explorer  (version)
 import qualified Pos.Client.CLI             as CLI
 import           Pos.Network.CLI            (NetworkConfigOpts, networkConfigOption)
-import           Pos.DHT.Model              (DHTKey)
-import           Pos.DHT.Real.CLI           (dhtExplicitInitialOption, dhtKeyOption,
-                                             dhtNetworkAddressOption, dhtPeersFileOption)
 import           Pos.Statistics             (EkgParams, StatsdParams, ekgParamsOption,
                                              statsdParamsOption)
 import           Pos.Util.BackupPhrase      (BackupPhrase, backupPhraseWordsNum)
-import           Pos.Util.TimeWarp          (NetworkAddress, addrParser)
 
 data Args = Args
```

Comment:
There is [this issue](https://issues.serokell.io/issue/CSL-1534), btw.

Reply:
It should just use `CommonNodeArgs` I believe, but ok, let's leave it for future.